assets/components/productPage/productPageTabs/productPageTabs.scss

```diff
@@ -13,24 +13,47 @@
 }
 
 .component-product-page-tabs__li {
+  overflow: hidden;
   @include mq($until:phablet) {
     flex: 1 1 0;
   }
 }
 
+.component-product-page-tabs__li:last-child {
+  box-shadow: 1px 0 0 0 gu-colour(garnett-neutral-4);
+}
+
+.component-product-page-tabs__li + .component-product-page-tabs__li .component-product-page-tabs__tab {
+  &:before {
+    position: absolute;
+    bottom: 1.0625rem;
+    left: 0;
+    width: 1px;
+    height: 100%;
+    background: gu-colour(garnett-neutral-4);
+    content: '';
+  }
+}
+
 .component-product-page-tabs__tab {
-  @include gu-fontset-heading-sans;
+  position: relative;
   text-decoration: none;
-  padding: ($gu-v-spacing)/1.25 $gu-h-spacing/2 ($gu-v-spacing)/1.5;
+  padding: ($gu-v-spacing)*1.3 $gu-h-spacing/2;
+  margin: 0 -1px 0 0;
   display: block;
   width: 100%;
   border: 0;
   margin-right: -1px;
-  box-shadow: inset 0 0 0 1px gu-colour(garnett-neutral-4); 
+  box-shadow: inset 0 1px 0 0 gu-colour(garnett-neutral-4);
   color: gu-colour(garnett-neutral-1);
   text-align: left;
   appearance: none;
   box-sizing: border-box;
+  transition: box-shadow 0.2s;
+  font-family: $gu-headline;
```

Comment:
can we turn this into a `@gu-fontset-` mixin?

Reply:
done!